src/base/settingsstorage.h

```diff
@@ -46,7 +46,7 @@ concept IsQFlags = std::same_as<T, QFlags<typename T::enum_type>>;
 
 // There are 2 ways for class `T` provide serialization support into `SettingsStorage`:
 // 1. If the `T` state is intended for users to edit (via a text editor), then
-//    implement `IStringable` interface
+//    implement `Stringable` concept
```

Comment:
I would still prefer to leave similar comments unchanged.

Reply:
Why, please explain.

Reply:
> Maybe I should revise it to: `` satisfy `Stringable` concept ``

Concept it is part of "interface" of those templates where it is used as constraint. It is bad idea to declare that some class satisfy some requirements. There are really so many possible requirements that can be satisfied by class... The class does not have to satisfy any requirements on purpose. It's some template that is instantiated in a certain way (or cannot be instantiated at all) depending on the used constraints.

Reply:
>It is bad idea to declare that some class satisfy some requirements.

I've said it [before](https://github.com/qbittorrent/qBittorrent/pull/19355#discussion_r1288895724), I intended it to be a Named Requirement.

>There are really so many possible requirements that can be satisfied by class... The class does not have to satisfy any requirements on purpose. It's some template that is instantiated in a certain way (or cannot be instantiated at all) depending on the used constraints.

You are out of context here. The comment/context we are discussing here is telling the reader how to do things: how to make some user-provided class `T` to work with `SettingsStorage`.
Therefore we need to give the exact direction/requirements. We are not talking about C++ language limitations or design issues whatsoever.

Reply:
> You are out of context here. The comment/context we are discussing here is telling the reader how to do things: how to make some user-provided class `T` to work with `SettingsStorage`.

Ok. I satisfied by its latest form.